apps/frontend/src/app/CreateOrder/CreateOrder.tsx

```diff
@@ -1,51 +1,124 @@
-import React, { FC, useState } from 'react';
+import React, { ChangeEvent, FC, useCallback, useRef, useState } from 'react';
 import { useForm } from 'react-hook-form';
 import { useHistory } from 'react-router-dom';
 
 import type { CreateOrderDto, OrderModel } from '@pasnik/api/data-transfer';
 import { useOrdersFacade } from '@pasnik/orders-data-access';
+import { Alert, Button, InputAdornment, Paper, TextField } from '@mui/material';
+import { formatter, validUrl } from '../../utils';
+
 export const CreateOrder: FC = () => {
   const { register, handleSubmit } = useForm();
   const { createOrder } = useOrdersFacade();
 
+  const [restaurant, setRestaurant] = useState<string>('');
```

Comment:
hmm react-hook-form should automatic handle the onChange and form state so you dont need to have to create controller for each input